app/src/main/res/values-ru/strings.xml

```diff
@@ -42,7 +45,9 @@
     <string name="credits_content3">Вы можете посмотреть исходный код приложения, сообщить о проблеме или предложить новые функции на <a href="https://github.com/Screaminghawk/android-slideshow">Github</a>.</string>
     <string name="credits_content4">Это приложение с открытым исходным кодом и оно полностью бесплатно; лицензировано в соответствии с <a href="https://tldrlegal.com/license/mit-license">MIT license</a>.</string>
     <string name="credits_content5">Благодарности  <a href="https://github.com/psyanite">Yan Tsui</a> за тестирование и дизайн иконки.</string>
-
+    <string name="credits_content6">Этот перевод &appname; предоставлен вам автором
+		<a href="https://michael.standen.link">&author;</a>.
```

Comment:
You should replace this with **your** name and website. You deserve credit for translating the app, so you'll have your name displayed on the credits page :)